Dockerfile

```diff
@@ -0,0 +1,21 @@
+ARG PYTHON_VERSION=3.8
+
+FROM python:${PYTHON_VERSION} AS base
+
+WORKDIR /app
+
+COPY . ./
+
+FROM python:${PYTHON_VERSION}-slim
+
+WORKDIR /app
+
+ENV PYTHONUNBUFFERED=1
+
+COPY --from=base /app .
+
+RUN pip install -r requirements.txt && python manage.py migrate

```

Comment:
Need to update pip before installation using command  `pip install --upgrade pip` and will be good to separate step of migration